runelite-client/src/main/java/net/runelite/client/plugins/timetracking/TimeTrackingConfig.java

```diff
@@ -39,14 +39,14 @@
 	String STOPWATCHES = "stopwatches";
 
 	@ConfigItem(
-		keyName = "estimateRelative",
-		name = "Show relative time",
-		description = "Show amount of time remaining instead of completion time",
+		keyName = "timeFormatMode",
+		name = "Display time as",
```

Comment:
A better name would be `Time format`

Reply:
That's much better. Honestly had no idea what to call this.